### PR TITLE
Fixes #16986: Update get-started for 6.0

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/16986

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: src/get-started/modules/ROOT/pages/advanced-configuration/apply.adoc [hunk 1]
```diff
@@ -5,16 +5,22 @@ To do so, got to the directives page (*Configuration management -> Directives*)
 and find our new technique (for example by using the filter with "Website"),
 and create a directive.
 
-This times our technique takes parameters, so we have to provide them:
+We will configure the site using the node property we defined in xref:get-started:node-management:data.adoc#file_from_template_mustache[the node management section].
 
-image::./technique-parameters.png["Parameters for our website technique", align="center"]
+The syntax to use to access them is `${node.properties[property_name]}`.
+Here we have hierarchical json, so we access a sub-key with `${node.properties[property_name][sub_key]}`.
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/advanced-configuration/apply.adoc]: Should we use `json` or `JSON` ?

File: src/get-started/modules/ROOT/pages/advanced-configuration/apply.adoc [hunk 2]
```diff
@@ -5,16 +5,22 @@ To do so, got to the directives page (*Configuration management -> Directives*)
...
+
+This times our technique takes parameters, so we have to provide them:
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/advanced-configuration/apply.adoc]: "This time"

File: src/get-started/modules/ROOT/pages/advanced-configuration/technique.adoc [hunk 3]
```diff
@@ -200,6 +204,12 @@ Conditions can be combinated using boolean operators:
 
 ====
 
+Let's add a *Service restart* method, with the `nginx` service name as parameter.
+
+Then use the *Conditions* tab of the method t add the repaired condition of the nginx configuration:
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/advanced-configuration/technique.adoc]: "to add"

File: src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc [hunk 4]
```diff
@@ -25,9 +25,9 @@ Let's use the filter in the directive tree to find it easily:
...
 image::./ssh.png["SSH technique", align="center"]
 
-Now click on "SSH server (OpenSSH)", the right part of the page will display the technique details, and
+Now click on *SSH server (OpenSSH)*, the right part of the page will display the technique details, and
 the list of available versions of the technique. We will use the latest one, by clicking on
-"Create with latest version".
+*Create with latest version*.
 
 This will make a new form appear, containing the configuration of the directive itself.
 As we said before, a technique is a parametrizable policy. A directive is a technique instance,
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc]: I'm not sure about "parametrizable"

File: src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc [hunk 5]
```diff
@@ -39,13 +39,14 @@ First, let's give it a description. When using Rudder, keep in mind that filling
 tedious, is very important for future reference, or collaboration with other.
 Is is for example useful to include reference to external information (ticketing system, etc.),
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc]: "**It** is for exemple

File: src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc [hunk 6]
```diff
@@ -39,13 +39,14 @@ First, let's give it a description. When using Rudder, keep in mind that filling
...
 to allow keeping a track of all configuration changes.
+Also note these documentation fields are rendered as markdown, so you can include links, bullet points, etc.
```
Comment (ElaadF) [inline src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc]: "Also, "

File: src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc (hunk 4 above)
Comment (amousset) [reply src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc]: I wasn't sure but it seems https://en.wiktionary.org/wiki/parametrizable

**Comment by amousset** [comment]
PR updated with a new commit

File: src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc (hunk 4 above)
Comment (ElaadF) [reply src/get-started/modules/ROOT/pages/configuration-policies/directive.adoc]: I couldn't find the word in the dictionary, that's why :joy:

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**